packages/front-end/tests/e2e/lab-manager/Labs_Lab_Manager.spec.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from 'playwright/test';
+
+test('01 - Create a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/labs`);
+  await page.waitForTimeout(2000);
+
+  // confirm button if enabled/visible
+  await expect(page.getByRole('button', { name: 'Create a new Lab' })).toBeHidden();
+});
+
+test('02 - Remove a Laboratory Unsuccessfully', async ({ page, baseURL }) => {
```

Comment:
Should this read "Successfully" instead of "Unsuccessfully"?

Reply:
This one is to check that user is not able to delete a Lab, it only checks for the 'remove option